### PR TITLE
Add Jalebi (YU Yunique)

### PR DESCRIPTION
**Pull request by hpnightowl**

### CODE REVIEW

**Comment by ghost** [comment]
Kindly PM @gwolf2u On Telegram.

**Comment by hpnightowl** [comment]
DONE THAT @Revanth He only said to pull request

File: update.xml [hunk 1]
```diff
@@ -736,6 +736,21 @@
             <forum>https://bit.ly/2P4aLmt</forum>
             <telegram>https://t.me/ghostrider_reborn</telegram>
         </lettuce>
+	    
+    <manufacturer id="Yu">
```
Comment (gwolf2u) [inline update.xml]: this can be removed as manufacturer "YU" has already been defined above (line 727)

File: update.xml [hunk 2]
```diff
@@ -736,6 +736,21 @@
...
+            <filename>crDroidAndroid-9.0-20190409-jalebi-v5.2</filename>
+            <buildtype>Monthly</buildtype>
+            <download>https://bit.ly/2VyziQI</download>
+            <changelog>https://bit.ly/2G7fX2z</changelog>
+            <gapps>http://opengapps.org</gapps>
+            <forum>https://bit.ly/2IooeBN</forum>
+            <modem>https://bit.ly/2G619ku</modem>
+            <telegram>https://t.me/hpnightowl</telegram>
+        </jalebi>
+
```
Comment (gwolf2u) [inline update.xml]: this blank line can also be removed

**Review by gwolf2u — COMMENTED** [review]
please correct based on my review

File: update.xml (hunk 2 above)
Comment (hpnightowl) [reply update.xml]: ok

File: update.xml (hunk 1 above)
Comment (hpnightowl) [reply update.xml]: ok

Comment (hpnightowl) [reply update.xml]: corrected

File: update.xml (hunk 2 above)
Comment (hpnightowl) [reply update.xml]: corrected

**Comment by adithya2306** [comment]
@hpnightowl you could've amended the existing commit with changes, then force pushed in your fork, it'd have reflected here .. anyway its not an issue as @gwolf2u can squash and merge in 1 commit

**Comment by hpnightowl** [comment]
> @hpnightowl you could've amended the existing commit with changes, then force pushed in your fork, it'd have reflected here .. anyway its not an issue as @gwolf2u can squash and merge in 1 commit

ok next time i will definately do that thanks i forget u told that ;p

File: update.xml [hunk 3]
```diff
@@ -753,7 +766,7 @@
 
         <z2_plus>
             <maintainer>Yashodhan Sawardekar</maintainer>
-            <devicename>ZUK Z2 Plus</devicename>
+￼             <devicename>ZUK Z2 Plus</devicename>
```
Comment (gwolf2u) [inline update.xml]: you messed around with this 
delete it

**Review by gwolf2u — CHANGES_REQUESTED** [review]
seems fine, just needs a small change on ZUK Z2 Plus (don't edit other devices)

File: update.xml (hunk 3 above)
Comment (hpnightowl) [reply update.xml]: sry next time i will use my pc :) to do all this done changes

File: update.xml [hunk 4]
```diff
@@ -753,7 +766,7 @@
 
         <z2_plus>
             <maintainer>Yashodhan Sawardekar</maintainer>
-            <devicename>ZUK Z2 Plus</devicename>
+￼            <devicename>ZUK Z2 Plus</devicename>
```
Comment (gwolf2u) [inline update.xml]: there's still an issue on this line 
not sure that char is this "￼ " just remove it

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: update.xml (hunk 4 above)
Comment (hpnightowl) [reply update.xml]: where i don't get u @gwolf2u

Comment (hpnightowl) [reply update.xml]: done @gwolf2u

Comment (hpnightowl) [reply update.xml]: sry for inconvenience next time i will use my pc :)

**Review by gwolf2u — APPROVED** [review]
check PM on Telegram

**Pull request closed without merging**